### PR TITLE
Ignore camera requests for a 100ms right after a request

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: loom/engine/bindings/loom/lmApplication.cpp [hunk 1]
```diff
@@ -511,6 +515,15 @@ static utArray<LoomApplicationGenericEventCallbackNote> gNativeGenericCallbacks;
...
 void LoomApplication::fireGenericEvent(const char *type, const char *payload)
 {
+    if (strcmp(type, "cameraRequest") == 0)
+    {
+        int currentTime = platform_getMilliseconds();
+        if (currentTime > lastCameraRequestTimestamp && currentTime - lastCameraRequestTimestamp < CAMERA_REQUEST_IGNORE_TIME)
+            return;
+    }
+
+    lastCameraRequestTimestamp = platform_getMilliseconds();
```
Comment (SmilyOrg) [inline loom/engine/bindings/loom/lmApplication.cpp]: Doesn't this have to be up in the conditional?

**Comment by SmilyOrg** [comment]
One nit!

File: loom/engine/bindings/loom/lmApplication.cpp (hunk 1 above)
Comment (themuflon) [inline loom/engine/bindings/loom/lmApplication.cpp]: Oh yeah, totally focused too much on possible overflows.

**Comment by SmilyOrg** [comment]
Seems fine now.

**Pull request merged**